testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeWriteDatabricksCompatibility.java

```diff
@@ -330,7 +331,7 @@ public void testInsertingIntoDatabricksTableWithAddedNotNullConstraint()
         }
     }
 
-    @Test(groups = {DELTA_LAKE_DATABRICKS, PROFILE_SPECIFIC_TESTS})
+    @Test(groups = {DELTA_LAKE_DATABRICKS_104, PROFILE_SPECIFIC_TESTS})
```

Comment:
Why is this change made?

Reply:
The connector doesn't support VACUUM with deletion vector, so I specified the old version here. Added a comment to revert this change in #22809.